Mathlib/Analysis/LocallyConvex/Basic.lean

```diff
@@ -190,11 +190,11 @@ theorem Balanced.smul_mem_mono [SMulCommClass 𝕝 𝕜 E] (hs : Balanced 𝕝 s
   rcases eq_or_ne a 0 with rfl | ha₀
   · simp_all
   · calc
-      b • x = (a⁻¹ • b) • a • x := by rw [smul_comm, smul_assoc, smul_inv_smul₀ ha₀]
-      _ ∈ s := by
+      (a⁻¹ • b) • a • x ∈ s := by
```

Comment:
This is more amusing than problematic in my mind. Since we swap the order into `mem`, the reading of the variables from the notation is reversed from how Lean sees them and `calc` relies on this to work in its current direction when using membership. I think its better than the status quo because 
- you can use subset instead of superset 
- it promotes complex -> simpler terms as a pattern